File: Box/app/src/main/java/com/danieladams/android/aca/box/MainActivity.java
package com.danieladams.android.aca.box;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        box bx = new box();
        bx.height = 50;
        bx.depth =20;
        bx.weight = 45;


        bx.showAttributes();

    }
}

File: Solitaire/app/src/main/java/com/danieladams/android/aca/solitaire/MainActivity.kt
package com.danieladams.android.aca.solitaire

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import org.jetbrains.anko.*

class MainActivity : AppCompatActivity() {

   override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)


        var counter = 0
        relativeLayout {
            val counterTextView = textView {
                id = 21
                text = "0"
                textSize = 24f
            }
            button {
                onClick {
                    counter++
                    counterTextView.text= counter.toString()
                }
            }.lparams {
                below(counterTextView)
            }
        }


    }
}
